presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -5877,21 +5877,21 @@ public void testCollectColumnStatisticsOnCreateTable()
                         "('c_bigint', null, 2.0E0, 0.5E0, null, '1', '2'), " +
                         "('c_double', null, 2.0E0, 0.5E0, null, '2.3', '3.3'), " +
                         "('c_timestamp', null, 2.0E0, 0.5E0, null, null, null), " +
-                        "('c_varchar', 8.0E0, 2.0E0, 0.5E0, null, null, null), " +
-                        "('c_varbinary', 8.0E0, null, 0.5E0, null, null, null), " +
+                        "('c_varchar', 16.0E0, 2.0E0, 0.5E0, null, null, null), " +
+                        "('c_varbinary', 16.0E0, null, 0.5E0, null, null, null), " +
```

Comment:
Why there is a change here?

Reply:
I don't know. I guess it is related to stats normalization.

Reply:
Will check it.

Reply:
I investigated this. It was because I haven't used non null values fraction when counting column data size. Fixed. Thanks!